decidim-core/db/migrate/20211126183540_add_timestamps_to_content_blocks.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddTimestampsToContentBlocks < ActiveRecord::Migration[6.0]
```

Comment:
Nice one! I didn't know that we didn't have the timestamps columns in `content_blocks` table. Although, are you using it for something?

Reply:
I do have some content blocks in futureEu that are static, and this is actually helping us to keep a cached version of the content block on the homepage.